graphix/random_objects.py

```diff
@@ -1,36 +1,46 @@
 from __future__ import annotations
 
+import functools
+from typing import TYPE_CHECKING
+
 import numpy as np
+import numpy.typing as npt
 import scipy.linalg
 from scipy.stats import unitary_group
 
 from graphix.channels import KrausChannel
 from graphix.ops import Ops
 from graphix.sim.density_matrix import DensityMatrix
+from graphix.transpiler import Circuit
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable, Iterator
+
+    from numpy.random import Generator
 
 
-def rand_herm(sz: int):
+def rand_herm(sz: int, rng: Generator) -> npt.NDArray:
```

Comment:
can you make `rng` optional? I don't think it's handy to set up rng all the time..

Reply:
Fixed in 33231b6. Thanks!